Firebase/InstanceID/FIRInstanceIDAuthKeyChain.m

```diff
@@ -115,9 +115,6 @@ - (NSMutableDictionary *)keychainQueryForService:(NSString *)service account:(NS
       }
     }
   }
-  if (passwordInfos != NULL) {
-    CFRelease(passwordInfos);
```

Comment:
Will this leak now?

Reply:
It does cause leak. Revert it and CFBridgingRelease fixes the issue and no memory leak.

http://www.arigrant.com/blog/2013/10/27/crossing-the-bridge-to-objective-c-arc